.github/workflows/slack_notify.yml

```diff
@@ -39,3 +39,12 @@ jobs:
           channel: github-p1
           color: red
           verbose: true
+      - uses: actions-ecosystem/action-slack-notifier@v1
+        if: ${{ github.event.label.name == 'C-triage' }}
+        with:
+          slack_token: ${{ secrets.SLACK_TOKEN }}
+          message: |
+            `${{ github.event.label.name }}` label has been added to "${{ github.event.issue.title }}" ([#${{ github.event.issue.number }}](https://github.com/MaterializeInc/materialize/issues/${{ github.event.issue.number }})).
+          channel: github-triage
+          color: red
+          verbose: false
```

Comment:
```suggestion
            `${{ github.event.label.name }}` label has been added to "${{ github.event.issue.title }}" (#${{ github.event.issue.number }}).
          channel: github-triage
          color: red
          verbose: true
```

Reply:
<img width="767" alt="Screenshot 2022-08-03 at 12 39 39" src="https://user-images.githubusercontent.com/23521087/182588821-1c3cbdbf-a7c1-4679-acbe-305037c7c5cf.png">

Just tested locally. Both `P1` and `C-Triage` seem to be triggered as should be! Patching a little copy paste error and setting verbose to `true` so you get a richer overview of the issue!

Reply:
I put this to false on purpose and directly linked to the issue. The messages in Slack are pretty... verbose... otherwise 😄. Happy to leave it at true if you prefer!

Reply:
Ah, then cool. I didn't really invest in making these look nice at the time, so whatever you prefer!

Reply:
Ok, I'll make both the same and merge. Thank you for testing! <3